x-pack/plugins/cases/public/components/files/files_table.test.tsx

```diff
@@ -0,0 +1,66 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React from 'react';
+import { screen } from '@testing-library/react';
+
+import { basicAttachment } from '../../containers/mock';
+import type { AppMockRenderer } from '../../common/mock';
+import { createAppMockRenderer, TestProviders } from '../../common/mock';
+import { FilesTable } from './files_table';
+
+const defaultProps = {
+  items: [basicAttachment],
+  pagination: { pageIndex: 0, pageSize: 10, totalItemCount: 1 },
+  onChange: jest.fn(),
+  isLoading: false,
+};
+
+describe('FilesTable', () => {
```

Comment:
Could you please add the following tests:

- Test the `resultsCount` text
- Test the add file button (not to test the upload just that it opens the file picker)
- Test the file preview (that opens/close correctly)